package.json

```diff
@@ -7,12 +7,13 @@
   "license": "GPLv3",
   "private": true,
   "dependencies": {
+    "@acala-network/chopsticks": "^0.4.1",
     "@babel/plugin-proposal-decorators": "^7.17.8",
     "@commitlint/config-conventional": "^17.4.2",
     "@mangata-finance/sdk": "1.16.1",
     "@types/bs58": "^4.0.1",
     "@types/jsonpath": "^0.2.0",
-    "@types/node": "^14.14.10",
+    "@types/node": "^18.14.6",
```

Comment:
does this mean that we need to bump node version everywhere?

Reply:
hmm, haven't noticed why it did upgrade, will try to revert